### ISSUE TITLE
Suppress warnings for setting particle sigma=1A

### ISSUE THREAD

**Issue by nathanmlim** [MobleyLab/blues#58]
This is just a suggestion to fix a minor annoyance.

Using the new refactored framework where we provide a `parmed.Structure` as input. There is an absurd amount of warning messages like:
>  particle 22339 has Lennard-Jones sigma = 0 (charge=0.417 e, sigma=0.0 nm, epsilon=0.0 kJ/mol); setting sigma=1A

This appears to be coming out of [openmm.alchemy](https://github.com/choderalab/openmmtools/blob/a2b3131b24ee5ab2dcc23baa4506a1b9b027e3a1/openmmtools/alchemy.py#L1507) but I've been told these warnings are safe to ignore. 

I'm not quite sure what the best way to address this is?

**Comment by nathanmlim** [comment]
Trying to investigate how to correct this a bit more as when trying to run the example in say an iPython notebook the excessive warnings cause lots of slowdown.

It seems there is some functionality to set the sigma factor to 1...found in this line [here](https://github.com/ParmEd/ParmEd/blob/master/parmed/openmm/parameters.py#L499) by setting the option `separate_ljforce` in the OpenMMParameterset to True. I'm not sure if this would have some deeper consequences to the OpenMM simulation. Comments on this option are noted [here](https://github.com/ParmEd/ParmEd/blob/master/parmed/openmm/parameters.py#L154)

Alternatively, I wonder if there is a way to suppress the warnings coming out of the `logger.warning()` function in [openmmtools.alchemy](https://github.com/ParmEd/ParmEd/blob/master/parmed/openmm/parameters.py#L154)

**Comment by nathanmlim** [comment]
Found a way to temporarily disable the warnings coming out of alchemy. Add these lines to change the logger level.

```python
import logging
logging.getLogger('openmmtools.alchemy').setLevel(logging.ERROR)
```

**Comment by nathanmlim** [comment]
Closing. Solved by PR #69
